src/App.css

```diff
@@ -1,77 +1,326 @@
+  height: 4.25rem;
+  border-style: solid;
+  border-width: 5px;
+}
+
+.percentage {
+  display: flex;
+  flex-direction: column;
+  font-family: 'Montserrat', sans-serif;
+  color: #121212;
+}
+
+.percentageNum {
+  width: 4.25rem;
+  height: 2.438rem;
+  margin: 0.063rem 5.75rem 0 0.688rem;
+  font-size: 2rem;
+}
+
+.completed {
+  width: 5rem;
+  height: 1.125rem;
+  margin: 0.438rem 4.938rem 0.75rem 0.75rem;
+  opacity: 0.5;
+  font-size: 0.875rem;
+}
+
+.bookChapter {
+  margin-right: 100px;
+  display: flex;
+  flex-direction: column;
+  font-family: 'Roboto Slab', serif;
+  margin-top: 2.063rem;
+}
+
+.currentChapter {
+  width: 7.875rem;
+  height: 1.125rem;
+  margin: 0.063rem 3.625rem 0 3.688rem;
+  color: #121212;
+  opacity: 0.5;
+  font-size: 0.813rem;
+  font-weight: 300;
+}
+
+.chapter {
+  width: 4.688rem;
+  height: 1.313rem;
+  margin: 0.438rem 6.813rem 0.25rem 3.688rem;
+  color: #121212;
+  font-size: 1rem;
+  font-weight: 300;
+}
+
+.progressBtn {
+  width: 11.5rem;
+  height: 2.063rem;
+  margin: 0.75rem 0 0.563rem 3.688rem;
+  padding: 0.438rem 1.188rem 0.5rem 1.375rem;
+  border-radius: 3px;
+  border: none;
+  background-color: #0290ff;
+  color: #fff;
+}
+
+.progressBtn:focus,
+.progressBtn:hover {
+  background-color: #fff;
+  color: #0290ff;
+  border: 3px solid #0290ff;
+  cursor: pointer;
 }
 
 .addCard {
-  margin: 48px;
-  padding: 16px 32px;
-  align-items: baseline;
-  border: 1px solid #d8d8d8;
-  border-radius: 4.8px;
-  gap: 5px;
-  background-color: #fafafa;
-}
-
-h2 {
-  color: #999898;
-  font-size: 20px;
-  font-weight: bold;
+  display: flex;
+  flex-direction: column;
+  border-top: solid 1px #e8e8e8;
+  margin: 1.813rem 0;
+  height: auto;
+}
+
+.formTitle {
+  height: 1.5rem;
+  margin: 1.813rem 32.75rem 1.188rem 6.25rem;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 1.25rem;
+  color: #888;
+}
+
+.AddNew {
+  display: flex;
+  margin: 1.833rem 6.25rem;
+  gap: 1rem;
+  margin-bottom: 15rem;
 }
 
-.addNew {
-  width: 75rem;
-  margin: 0 0.063rem 0.938rem 0;
-  padding: 2rem 7.438rem 1.625rem 1.688rem;
+input {
+  font-family: 'Montserrat', sans-serif;
+  font-size: 1rem;
+  color: #c4c4c4;
+  height: 1.188rem;
+  letter-spacing: -0.15px;
+  padding: 0.813rem 8.688rem 0.813rem 1.063rem;
   border-radius: 4px;
-  border: solid 1px #e8e8e8;
+  border: 1px solid #c4c4c4;
   background-color: #fff;
 }
 
-.navbar {
-  display: flex;
-  width: 100%;
-  gap: 10%;
-  margin: 48px;
+select {
+  height: 2.813rem;
+  letter-spacing: -0.15px;
+  padding: 0.813rem 8.688rem 0.813rem 1.063rem;
+  border-radius: 4px;
+  border: 1px solid #c4c4c4;
+  background-color: #fff;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 1rem;
+  color: #c4c4c4;
 }
 
-.link {
+.addBtn {
+  background-color: #0290ff;
+  border-radius: 3px;
+  border: none;
+  width: 7rem;
+  height: 2.813rem;
+  color: #fff;
+  text-align: center;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 1rem;
+  right: 10%;
+}
```

Comment:
- [ ] Hi, the project requirements says your design should match Zeplin's design include the same styling.
- I would suggest you set `width: 100%` instead of `7rem` because `Add Book` button is not responsive Kindly pay attention to the red sign on the pictures to easily understand what I mean. 
- Can you increase a little bit the font color of `Book Title`, `Book Author`, `Action`. This will help the user to see what he is typing without forcing the views.
- Your design bellow:
![Capture](https://user-images.githubusercontent.com/63972670/230334502-565d4566-ea7b-4975-81cd-4b841a854c2d.PNG)


- Original Design bellow:

![Capture1](https://user-images.githubusercontent.com/63972670/230333313-1dc8a760-7a6b-48e0-96b2-dc6918af79c8.PNG)